### PR TITLE
#1397 Wrong Packet Id of MySQL Protocol Packet If Error Happens In Pr…

### PR DESCRIPTION
**Pull request by ssxlulu**
…ocess of Loading Data

Reason:  
  BUG.  
Type:  
  BUG
Influences：

### CODE REVIEW

File: src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java [hunk 1]
```diff
@@ -208,7 +208,7 @@ public void start(String strSql) {
                 clear();
                 serverConnection.writeErrMessage(ErrorCode.ER_FILE_NOT_FOUND, msg);
             } else {
-                if (parseFileByLine(fileName, loadData.getCharset(), loadData.getLineTerminatedBy())) {
+                if (parseFileByLine((byte) 0, fileName, loadData.getCharset(), loadData.getLineTerminatedBy())) {
```
Comment (sunsun314) [inline src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java]: input value 0 is improper when MultiQueries would be execute like  
"use db1;load data infile '/opt/test.txt' into table test2"

Comment (ssxlulu) [reply src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java]: As I know,DBLE doesn't truly support the MultiQueries.

As MySQL offical document says about MySQL protocol:
> The sequence-id is incremented with each packet and may wrap around. It starts at 0 and is reset to 0 when a new command begins in the Command Phase.

When a new command begins in the Command Phase, the sequence id will be reset to 0.

Comment (ssxlulu) [reply src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java]: What's more, test must be execute to verfy the theory.

Comment (sunsun314) [reply src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java]: dble has a problem in load data multiQueries. I have created a new issue for the problem https://github.com/actiontech/dble/issues/1403
You can pay attention to or try to fix this problem if you are interested

Comment (ssxlulu) [reply src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java]: Okay,Resolve this problem for simpler scenario first, and i will pay attention to that problem.

**Pull request merged**